aop/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+module micronaut.aop {
+    requires transitive micronaut.core;
+    requires static micronaut.core.reactive;
+    requires static kotlin.stdlib;
```

Comment:
why is Kotlin required? There is no requirement for Kotlin to be on the class path in a non-modular scenario?

Reply:
In a non modular scenario this file is ignored. When you use the jar on the classpath it is just a jar with classes. Module-info.java is only used when a its on the Module-path (eg. when used in modular context)

`requires static` also makes it a compile only dependency which is what we need - there references to kotlin
the dependency is from `build.gradle` -> `compileOnly libs.managed.kotlinx.coroutines.core`